design-reviews/trade-studies/readme.md

```diff
@@ -0,0 +1,47 @@
+# Trade Studies
+
+Trade studies are a tool for selecting the best option out of several possible options for a given problem (for example: compute, storage).
+They evaluate potential choices against a set of objective criteria/customer requirements to clearly lay out the benefits and limitations
+of each solution.
+
+[Trade studies](https://en.wikipedia.org/wiki/Trade_study) are a concept from systems engineering that we adapted for CSE projects. Trade
+studies have proved to be a critical tool to drive alignment with the customers, earn credibility while doing so and ensure our decisions
+were backed by data and not bias.  
+
+## When to use the tool
+
+The ideal stage to do trade studies is after the game plan completion and before coding begins; they go hand in hand with defining high
```

Comment:
You might want to express this independent of CSE processes - i.e. after the project has been scoped and before coding for a specific story begins. I'd also see trade studies as a tool that can be used throughout the engagement (as a result of spikes, or whenever there are multiple options).

Reply:
Good point. I just made the update